semeio/jobs/scripts/ots.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+from semeio.jobs.ots.ots import ots_run
+from semeio.jobs.ots.ots_config import generate_rst_doc
+import argparse
+from semeio import valid_file
+
+description = (
+    "Overburden timeshift (OTS) generates evolution of reservoir surfaces"
+    "based on eclipse models and seismic velocity volume. If the shift"
+    "becomes negative it won't be nullified but only divided"
```

Comment:
Missing space between shift and becomes

Reply:
And a few other lines come to think of it

Reply:
Might be, as wasn't really sure if the space is automatically added or not -  so apparently not :)